### DISCUSSION TITLE
Shelly Plus RGBW PM for tunable white LED and connected wall switch

### DISCUSSION THREAD

**Discussion by kaahla** [mongoose-os-apps/shelly-homekit#1582] [Q&A] [answered]
Hello together,
I am using my shelly Plus RGBW PM in the CCT mode with two connected tunable white LED stripes (2 channels: cold white and warm white). This actually works great. My problem is: I want to use my two **wall switches** to control the LEDs. I connected them with "**I1**" and "**I3**", but the shelly is not reacting on the wall switch at all. Can somebody tell me what to do to get my wall switches working?
thank you a lot for your help :)

**Comment by markirb** [comment, marked as answer]
See here for an answer:

https://github.com/mongoose-os-apps/shelly-homekit/issues/1566

**Reply by kaahla** [reply]
thanks for your answer. The Shelly is working now, but only ONE wallswitch is working. how can I connect two switches to the same shelly RGBW?
